ts/Series/AreaRange/AreaRangeSeries.ts

```diff
@@ -284,14 +286,28 @@ class AreaRangeSeries extends AreaSeries {
                 point.plotY = null as any;
             } else {
                 point.plotLow = plotY as any;
-                point.plotHigh = yAxis.translate(
-                    series.dataModify ?
-                        series.dataModify.modifyValue(high) : high,
-                    0 as any,
-                    1 as any,
-                    0 as any,
-                    1 as any
-                );
+
+                // Calculate plotHigh value based on each yAxis scale (#15752)
+                if (chart.hasParallelCoordinates) {
+                    point.plotHigh = chart.yAxis[i].translate(
+                        series.dataModify ?
+                            series.dataModify.modifyValue(high) : high,
+                        0 as any,
+                        1 as any,
+                        0 as any,
+                        1 as any
+                    );
+                } else {
+                    point.plotHigh = yAxis.translate(
+                        series.dataModify ?
+                            series.dataModify.modifyValue(high) : high,
+                        0 as any,
+                        1 as any,
+                        0 as any,
+                        1 as any
+                    );
+                }
```

Comment:
Repeated code. The only difference here is `yAxis` used, so I think you could simplify it to a simple variable:
```js
const yAxis = isParallelCor ? chart.yAxis[i] : series.yAxis;
```